src/web/app/src/components/AboutPage.tsx

```diff
@@ -107,7 +107,17 @@ const AboutPage = () => {
         you want to see it..
       </p>
 
-      <h2>What is a Planet?</h2>
+      <h2>What does Telescope do?</h2>
+      <p>
+        In essence, Telescope takes RSS blog post feeds, rebuilds those blog posts into HTML and
```

Comment:
Can you add a like for `RSS` to https://en.wikipedia.org/wiki/RSS and maybe https://rss.com/blog/rss-feed-for-blog/ for "blog post feed"?

Reply:
Yes.